src/edu/csus/ecs/pc2/ui/TestResultsFrame.java

```diff
@@ -102,5 +102,18 @@ public void setValidatorOutputFileNames(String[] filenames) {
         getMultiTestSetOutputViewerPane().setValidatorOutputFileNames(filenames);
     }
 
+    /**
+     * Enables/Disables Execute All button
+    */
+    public void setExecuteAllButton(boolean b) {
+        getMultiTestSetOutputViewerPane().setExecuteAllButton(b);
```

Comment:
Similar to the above comment, I think method `setExecuteAllButton()` (in class `TestResultsPane`) should be renamed to reflect the fact that what it is doing is managing the _enabled state_ of the button.